Tests/DatadogTests/Datadog/Mocks/RUMFeatureMocks.swift

```diff
@@ -150,6 +150,24 @@ extension RUMEventsMapper {
     }
 }
 
+extension RUMSessionSampler {
+    static func mockAny() -> RUMSessionSampler {
```

Comment:
Quick `codestyle` question, why not using computed properties? Isn't it more elegant to use sth like
`.mockAny` instead of `.mockAny()` 🙌

Reply:
Next to `.mockAny()` (with no parameters) we also have `.mockWith(param1:param2:...)` convention. With static function semantic we can achieve similarity in both 🙂.